server/platform/services/searchengine/searchengine.go

```diff
@@ -45,8 +45,19 @@ func (seb *Broker) GetActiveEngines() []SearchEngineInterface {
 	if seb.ElasticsearchEngine != nil && seb.ElasticsearchEngine.IsActive() {
```

Comment:
Can we add the `IsIndexingEnabled` check for Elasticsearch as well? If I enable ES search and disable DB search, and at MM server startup ES server is not running, MM isn't able to search anywhere but I see `elasticsearch` in the admin console.

Reply:
Currently they mean the same thing:

```
func (es *ElasticsearchInterfaceImpl) IsActive() bool {
	return *es.Platform.Config().ElasticsearchSettings.EnableIndexing
}

func (es *ElasticsearchInterfaceImpl) IsIndexingEnabled() bool {
	return *es.Platform.Config().ElasticsearchSettings.EnableIndexing
}
```

But I think what you mean is to check that the client is initialized. Let me make that change.